lib/routes/chainswap/send/fee/fee_option.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:flutter_breez_liquid/flutter_breez_liquid.dart';
+
+extension PreparePayOnchainResponseAffordable on PreparePayOnchainResponse {
+  bool isAffordable({int? balance, int? walletBalance, required int amountSat}) {
```

Comment:
Are `walletBalance` and `amountSat` needed?

Reply:
They are not, removed.